handler.js

```diff
@@ -10,19 +10,24 @@ module.exports.scanOrg = async event => {
   assert(event.githubToken, 'No githubToken provided (required to check for vulnerabilities)')
   assert(process.env.RESULTS_S3_BUCKET_PREFIX, 'No RESULTS_S3_BUCKET_PREFIX provided')
   assert(process.env.STAGE, 'No STAGE provided')
-  
-  const vulnerabilitiesByRepo = await getSecurityVulnerabilitiesForMultiple({
-    bitbucket: {
+
+  let config = {
+    versionsCsvUrl: event.versionsCsvUrl
+  }
+  if (event.bitbucketAppPassword) {
+    config.bitbucket = {
       username: event.bitbucketUsername,
       appPassword: event.bitbucketAppPassword,
       workspace: event.bitbucketWorkspace,
-    },
-    github: {
+    }
+  }
+  if (event.githubToken) {
```

Comment:
It might be better to change this to check for a githubOrg, rather than a token, because the GitHub token is needed even when scanning Bitbucket repos. (The API for seeing if there are any vulnerabilities for a given dependency is a GitHub API, so the GitHub token is needed either way. See line 10.)
```suggestion
  if (event.githubOrg) {
```

I don't know whether it would also be good to change the Bitbucket condition to similarly be a check for a bitbucketWorkspace, for consistency. That seems like it could go either way.

**Edit:** Checking for a GitHub organization or Bitbucket workspace would also align with the way the vulernability-scanner itself determines what to check:
https://github.com/silinternational/vulnerability-scanner/blob/e9694e085d35fa1448e79f07ede2f2140a3fa676/index.js#L81-L102

Reply:
Good observation, thank you.